src/parser/content-parser.php

```diff
@@ -98,9 +116,13 @@ public function parse( $post_content, $post_id = null ) {
 	 *
 	 * @return array[string]array
 	 */
-	protected function source_block( $block, $registered_blocks ) {
+	protected function source_block( $block, $registered_blocks, $filter_options ) {
```

Comment:
Minor: Since we're changing this function to return `null` sometimes, we should include that in the PHPDoc:

```php
/*
 * ...
 * @return array[string]array|null
 */
```